recipes/squashfs/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "squashfs-tools" %}
+{% set version = "4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://downloads.sourceforge.net/project/squashfs/squashfs/squashfs{{ version }}/squashfs{{ version }}.tar.gz
+  sha256: 0d605512437b1eb800b4736791559295ee5f60177e102e4d4ccd0ee241a5f3f6
+  patches:
+    - 0001-modern-compression.diff
+
+build:
+  skip: True  # [not linux]
+  number: 0
+  script:
+    - pushd squashfs-tools
+    - make install
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - lz4-c
+    - lzo
+    - xz
+    - zlib
```

Comment:
Should these also be listed in `run`?

Reply:
It looks like most of those packages already have run_exports in them

  - zlib: https://github.com/conda-forge/zlib-feedstock/blob/58550035ead47ad3e3e45be33d37fbd9965a5cfa/recipe/meta.yaml#L14-L17
  - xz: https://github.com/conda-forge/xz-feedstock/blob/69aecee111a1a0653e21458b88888421e3ae2f34/recipe/meta.yaml#L16-L19
  - lzo: https://github.com/conda-forge/lzo-feedstock/blob/c6fe241568dcaa3226d5b9633d4b2c45ea814b96/recipe/meta.yaml#L14-L16
  - lz4-c: after conda-forge/lz4-c-feedstock#15 is merged.